app/uk/gov/hmrc/eoricommoncomponent/frontend/views/vat_details.scala.html

```diff
@@ -24,11 +24,15 @@
 @import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
 @import uk.gov.hmrc.govukfrontend.views.Aliases.Button
 @import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+@import uk.gov.hmrc.eoricommoncomponent.frontend.config.AppConfig
 
-@this(inputDate_di: inputDate, layout_di: layout, govukButton: GovukButton, inputText: helpers.inputText, errorSummary: helpers.errorSummary)
+@this(inputDate_di: inputDate, layout_di: layout, govukButton: GovukButton, inputText: helpers.inputText, errorSummary: helpers.errorSummary, appConfig:AppConfig)
 
 @(vatForm: Form[VatDetails], isInReviewMode: Boolean, service: Service)(implicit request: Request[_], messages: Messages)
 
+@vatUKLink = {
+<a class="govuk-link" href="@{appConfig.vatUKDetails}" target="_blank" rel="noopener noreferrer">@messages("cds.subscription.vat-details.intro-text.link")</a>
+}
 
```

Comment:
Can u add the test for view for the new content

Reply:
I have added the tests for this view. Please review.